### PR TITLE
[v6r19] System test for Transformation System using meta data filters

### PR DESCRIPTION
**Pull request by arrabito**
BEGINRELEASENOTES
Added  a system test for transformations with meta-filters.

Please follow the template:
*tests
FIX: Minor fixes in the TS system test
CHANGE: Add to the TS system test, the test for transformations with meta-filters 

ENDRELEASENOTES

### CODE REVIEW

File: TransformationSystem/scripts/dirac-transformation-get-files.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+tc = TransformationClient()
+res = tc.getTransformationFiles( {'TransformationID': TransID} )
+
+if not res['OK']:
+  DIRAC.gLogger.error ( res['Message'] )
+  DIRAC.exit( -1 )
+
+for transfile in res['Value']:
+  print transfile['LFN']
```
Comment (atsareg) [inline TransformationSystem/scripts/dirac-transformation-get-files.py]: please use gLogger.notice to print user destinated output in a command

Comment (arrabito) [inline TransformationSystem/scripts/dirac-transformation-get-files.py]: Just done.

**Comment by fstagni** [comment]
It got messed up with several other changes not from you. I'd suggest you open another PR from a fresh branch where you simply cherry-pick your commits.

**Comment by arrabito** [comment]
OK just done:
https://github.com/DIRACGrid/DIRAC/pull/3428

**Pull request closed without merging**